### ISSUE TITLE
Legacy App issue

### ISSUE THREAD

**Issue by hazelshapiro** [nsidc/usaon-benefit-tool#315]
Good to fix/address this if it's quick since we're still using the legacy app for demonstration purposes. 

[Index of Datasets](https://docs.google.com/spreadsheets/d/1XcAAHbZilAM9KEBshqjmretLwSFnCsd5OmdzJ7mQoEw/edit?gid=0#gid=0) 
[README doc](https://docs.google.com/document/d/1bWf9tHUJkXVHJ0s5a3PskbKjXCly-htqVMEOOIquvxQ/edit)
[New spreadsheet I am trying to add](https://docs.google.com/spreadsheets/d/1envoTaQE59sGkgnZJU8L6KMICrjHZAOGsh6f998JGbI/edit?gid=0#gid=0)

It looks to me like Google changed the format of it's links - I can't get the new spreadsheet ID in a format where it starts with 2PACX also where it used to say "/spreadsheets/d/e/" the URL is now just "/spreadsheets/d/"

I think that's why I am getting a "No Links" error.

**Comment by mfisher87** [comment]
@hazelshapiro the sheet is published, correct?

**Comment by hazelshapiro** [comment]
Yes, this is the published url https://docs.google.com/spreadsheets/d/1envoTaQE59sGkgnZJU8L6KMICrjHZAOGsh6f998JGbI/pubhtml

**Comment by mfisher87** [comment]
Thanks! [It's not the first time they've done this](https://stackoverflow.com/questions/40740118/has-the-google-sheets-published-url-suddenly-changed-to-a-different-format). How inconsiderate.

We should be saving the full URL in our index sheet, and updating the code to use that full URL instead of templating based on the old URL structure. The change sounds minor, but it's been some time since we deployed this app, and we may have taken shortcuts because we only expected to deploy it once, so there may be additional minor challenges there. What do you think @rmarow ? We have that deployment set up in this private repo, but I don't remember if you worked on it or not: https://github.com/nsidc/usaon-vta-webapp-legacy

**Comment by rmarow** [comment]
@MattF-NSIDC I don't recall doing any deploying of the legacy app during my time on this project besides reviewing it for work on the new app. Very frustrating that google changes the URL!
